src/lib.rs

```diff
@@ -1,18 +1,18 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- *  * License, v. 2.0. If a copy of the MPL was not distributed with this
- *   * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#[cfg(all(target_os = "linux", feature = "bluetooth"))]
-extern crate blurz;
-#[cfg(all(target_os = "android", feature = "bluetooth"))]
-extern crate blurdroid;
-#[cfg(all(target_os = "macos", feature = "bluetooth"))]
-extern crate blurmac;
-#[cfg(feature = "bluetooth-test")]
-extern crate blurmock;
-
-pub mod bluetooth;
-#[cfg(not(any(all(target_os = "linux", feature = "bluetooth"),
-              all(target_os = "android", feature = "bluetooth"),
-              all(target_os = "macos", feature = "bluetooth"))))]
-mod empty;
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *   * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+

+#[cfg(all(target_os = "linux", feature = "bluetooth"))]
+extern crate blurz;
+#[cfg(all(target_os = "android", feature = "bluetooth"))]
+extern crate blurdroid;
+#[cfg(all(target_os = "macos", feature = "bluetooth"))]
+extern crate blurmac;
+#[cfg(feature = "bluetooth-test")]
+extern crate blurmock;
+pub mod adapter;
+pub mod bluetooth;

```

Comment:
Be sure to remove the old BluetoothAdapter enum-based code from bluetooth.rs as well.

Reply:
I removed enum type BluetoothAdapter part from bluetooth.rs file. Some of the functions throw errors because one of its parameters is 'adapter : BluetoothAdapter' and the dependencies of BluetoothAdapter don't exist in this file anymore. I tried importing BluetoothAdapter from adapter.rs file using 'adapter::BluetoothAdapter' but that didn't work.
I'm getting the following error:
error[E0277]: the size for values of type `(dyn adapter::BluetoothAdapter + 'static)` cannot be known at compilation time
I have attached the corresponding line of code:
https://github.com/NivedithaShankar/devices/blob/master/src/bluetooth.rs#L304

Could you give any suggestion to fix this error?

Reply:
[This previous review comment](https://github.com/servo/devices/pull/32#discussion_r271111392) addressed this issue. Let me know if you have any questions about it.